src/arc.rs

```diff
@@ -206,3 +221,24 @@ impl Mul<Arc> for Affine {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[test]
+    fn flipped_arc() {
+        let a = Arc::new((0., 0.), (1., 0.), 0., PI, 0.);
+        let f = a.flipped();
+
+        // Most fields should be unchanged:
+        assert_eq!(a.center, f.center);
+        assert_eq!(a.radii, f.radii);
+        assert_eq!(a.x_rotation, f.x_rotation);
+
+        // Sweep angle should be in reverse
+        assert_eq!(a.sweep_angle, -f.sweep_angle);
+
+        // Flipping it again should result in the original arc
+        assert_eq!(a, f.flipped());
```

Comment:
No fuzzy testing needed, since there is only addition/subtraction meaning perfect accuracy, at least for normal input.

Reply:
You are confirming what I have here is fine, not suggesting a change, right?